@kiva/kv-components/vue/KvLendCta.vue

```diff
@@ -261,17 +273,20 @@ export default {
 		},
 		ctaButtonText() {
 			if (this.isCompleteLoanActive) {
-				return 'Lend';
+				return this.primaryButtonText || 'Lend';
 			}
 			switch (this.state) {
 				case 'loading':
 					return 'Loading...';
 				case 'refunded':
 				case 'expired':
 				default:
-					return 'Lend';
+					return this.primaryButtonText || 'Lend';
 			}
 		},
+		cta2ButtonText() {
```

Comment:
It might be clearer to rename this and the `secondaryButton` usage to be specifically `checkoutButton`, since that's what we are using that CTA for.

Reply:
I agree this naming is getting a little wonky...generally ok with this for now but I wonder if we should break it down by "stages" at somepoint... ie. ctaButtonText > stageOneButtonText + cta2ButtonText > stageTwoButtonText or something along those lines... or maybe lendButtonText and loanReservedButtonText...

Reply:
What about loanInBasketButtonText?

Reply:
ok done